### ISSUE TITLE
Keyboard navigation not working with videos

### ISSUE THREAD

**Issue by accforgithubtest** [glowinthedark/caddy-file-server-browse-extension#2]
This is awesome, thank you for creating this !

In a folder with many images and videos, a few videos are in a format that cannot be played by the browser.
When such a video is encountered, the keyboard navigation stops working, I cannot skip to next image/video. Seeing this issue on the latest firefox version on ubuntu dekstop.
It would be great if I am able to continue navigating even when a unsupported video / file is encountered.

**Comment by accforgithubtest** [comment]
Since issue #3 has been closed, I have noticed that this issue is not only when there are unsupported videos, but it looks like keyboard navigation does not work for all videos (even where the videos is in a browser supported format).
I have updated the issue title to reflect the same.

**Comment by glowinthedark** [comment]
For videos left/right arrows are handled as a special case and are intended to work as jump back/forward by 5 seconds _if_ the video is playing — specifically, when the video is playing, this behaviour is supposed to mimic the way left/right keys work on youtube (i.e. as short +/- jump), __but__ if the video is stopped or finished playing then pressing right arrow key is supposed to work as jump to next item in the list. 

At least this is the intended logic, and whether it works as expected or not might depend on which browser is used, for example, chrome adds some extra logic compared to ff, when handling spacebar and left/right key presses and this can interfere and mess up the currently naive logic in the template, also the behaviour can be different depending on which element currently has focus, which is another issue that is currently on my defects TODO list, with no clear approach yet (maybe will need to reverse engineer what youtube and other players are doing). By all means, thanks for raising it as it confirms it is something that needs addressing and fixing.
